spring-cloud-gcp-data-firestore/src/test/java/org/springframework/cloud/gcp/data/firestore/it/FirestoreRepositoryIntegrationTests.java

```diff
@@ -166,6 +168,73 @@ public void pageableQueryTest() {
 		assertThat(pagedUsers).containsExactlyInAnyOrder("blah-person5", "blah-person6");
 	}
 
+	@Test
+	public void inFilterQueryTest() {
+		User u1 = new User("Cloud", 22);
+		User u2 = new User("Squall", 17);
+		Flux<User> users = Flux.fromArray(new User[] {u1, u2});
+
+		this.userRepository.saveAll(users).blockLast();
+
+		List<String> pagedUsers = this.userRepository.findByAgeIn(Arrays.asList(22, 23, 24))
+				.map(User::getName)
+				.collectList()
+				.block();
+
+		assertThat(pagedUsers).containsExactly("Cloud");
+
+		pagedUsers = this.userRepository.findByAgeIn(Arrays.asList(17, 22))
```

Comment:
Does not need to be for this PR, but using StepVerifier makes it easier to consume and verify a stream in one sentence. Then you don't need to block or save an intermediate variable, can just do `.expectNext(...).verifyComplete()`.